net/http/inc/THttpWSHandler.h

```diff
@@ -0,0 +1,63 @@
+// $Id$
+// Author: Sergey Linev   20/10/2017
+
+/*************************************************************************
+ * Copyright (C) 1995-2017, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#ifndef ROOT_THttpWSHandler
+#define ROOT_THttpWSHandler
+
+#include "TNamed.h"
+
+#include "TList.h"
+
+class THttpCallArg;
+class THttpWSEngine;
+class THttpServer;
+
+class THttpWSHandler : public TNamed {
+
+friend class THttpServer;
+
+private:
+
+   THttpWSEngine *FindEngine(UInt_t id) const;
+
+   Bool_t DirecltyHandle(THttpCallArg *arg);
```

Comment:
Shouldn't it be DirectlyHandle? (BTW, I don't really like this name...)

Reply:
Yes, you are right - it was misspelling of DirectlyHandle. Now I call it HandleWS

Reply:
And anyway - it is private function, which is not accessible by the end-user

Reply:
OK, thanks for changing it (and private or not, I just didn't like it...) :-)